neuralmonkey/encoders/sentence_cnn_encoder.py

```diff
@@ -4,6 +4,8 @@
 from typeguard import check_argument_types
 
 from neuralmonkey.encoders.attentive import Attentive
+from neuralmonkey.encoders.sentence_encoder import RNNCellTuple
+from neuralmonkey.encoders.sentence_encoder import SentenceEncoder
```

Comment:
Padalo mi to v testech u toho parent encoderu, uz jsem to prepsal. Co se tyce predavani konvolucnich a highway siti, tak si nemyslim, ze by to melo byt ocekavane chovani. Spis si myslim, ze by mohla byt fajn moznost si vybrat, jestli je chci prevzit, nebo si chci vytvorit vlastni. Prece jen, ani tu RNN sit si prece nepredavas.